Lesson33/Gokhan/src/App.js

```diff
@@ -0,0 +1,76 @@
+import { List } from './components/List';
+import { Navbar } from './components/Navbar';
+import { Account } from './components/Account';
+import { PerformanceState } from './components/PerformanceState';
+import './App.css';
+import { loggedInUser, loggedOutUser, todoData } from './data';
+import { useState } from 'react';
+
+export const App = () => {
+	const [user, setUser] = useState(loggedInUser);
```

Comment:
Only user UserContext to store user data, no need to keep track of it locally in the component.